### DISCUSSION TITLE
timing is not work properly

### DISCUSSION THREAD

**Discussion by murdocklawless** [WarmUpTill/SceneSwitcher#717] [General]
timing is not working properly, rarely it's working. my B scene should turn back to A scene after 3 seconds but generally stuck on the B scene.

is there any way to fix this?

**Comment by WarmUpTill** [comment]
I am sorry to hear you are experiencing issues with the plugin!

Can you please share more details on what you have configured?
(Either screenshots or by exporting the settings to a file and sharing that)

Can you also try to reproduce the issue with verbose logging enabled (General tab) and share the corresponding log file?

**Comment by murdocklawless** [comment]
thanks for the reply, I'm using obs 29.0.2 64 bit version at windows 11 22H2, advanded scene switcher 1.21.1

here is screenshots,

https://i.imgur.com/WAELsua.png
https://i.imgur.com/MKUJ6hP.png

here is log file,

02:53:24.671: CPU Name: AMD Ryzen 9 5900X 12-Core Processor            
02:53:24.671: CPU Speed: 3693MHz
02:53:24.671: Physical Cores: 12, Logical Cores: 24
02:53:24.671: Physical Memory: 65442MB Total, 56774MB Free
02:53:24.671: Windows Version: 10.0 Build 22621 (release: 22H2; revision: 1555; 64-bit)
02:53:24.671: Running as administrator: false
02:53:24.671: Windows 10/11 Gaming Features:
02:53:24.671: 	Game DVR: On
02:53:24.671: 	Game Mode: Probably On (no reg key set)
02:53:24.671: 	Hardware GPU Scheduler: Probably On (no reg key set)
02:53:24.673: Sec. Software Status:
02:53:24.673: 	Microsoft Defender Virüsten Koruma: enabled (AV)
02:53:24.673: 
02:53:24.674: Current Date/Time: 2023-04-20, 02:53:24
02:53:24.674: Browser Hardware Acceleration: false
02:53:24.674: Hide OBS windows from screen capture: false
02:53:24.674: Qt Version: 6.3.1 (runtime), 6.3.1 (compiled)
02:53:24.674: Portable mode: false
02:53:24.825: OBS 29.0.2 (64-bit, windows)
02:53:24.825: ---------------------------------
02:53:24.826: ---------------------------------
02:53:24.826: audio settings reset:
02:53:24.826: 	samples per sec: 48000
02:53:24.826: 	speakers:        2
02:53:24.826: 	max buffering:   960 milliseconds
02:53:24.826: 	buffering type:  dynamically increasing
02:53:24.826: ---------------------------------
02:53:24.826: Initializing D3D11...
02:53:24.826: Available Video Adapters: 
02:53:24.829: 	Adapter 0: AMD Radeon RX 6900 XT
02:53:24.829: 	  Dedicated VRAM: 4245905408
02:53:24.829: 	  Shared VRAM:    4246081536
02:53:24.829: 	  PCI ID:         1002:73bf
02:53:24.829: 	  Driver Version: 31.0.14043.7000
02:53:24.829: 	  output 0:
02:53:24.829: 	    name=24G2W1G4
02:53:24.829: 	    pos={0, 0}
02:53:24.829: 	    size={1920, 1080}
02:53:24.829: 	    attached=true
02:53:24.829: 	    refresh=144
02:53:24.829: 	    bits_per_color=8
02:53:24.829: 	    space=RGB_FULL_G22_NONE_P709
02:53:24.829: 	    sdr_white_nits=80
02:53:24.829: 	    nit_range=[min=0.500000, max=270.000000, max_full_frame=270.000000]
02:53:24.831: Loading up D3D11 on adapter AMD Radeon RX 6900 XT (0)
02:53:24.870: D3D11 loaded successfully, feature level used: b000
02:53:24.870: DXGI increase maximum frame latency success
02:53:24.870: D3D11 GPU priority setup failed (not admin?)
02:53:25.941: ---------------------------------
02:53:25.941: video settings reset:
02:53:25.941: 	base resolution:   1920x1080
02:53:25.941: 	output resolution: 1920x1080
02:53:25.941: 	downscale filter:  Bicubic
02:53:25.941: 	fps:               60/1
02:53:25.941: 	format:            NV12
02:53:25.941: 	YUV mode:          Rec. 709/Partial
02:53:25.941: NV12 texture support enabled
02:53:25.941: P010 texture support not available
02:53:25.943: Audio monitoring device:
02:53:25.943: 	name: HoparlÃ¶r (Logitech PRO X Gaming Headset)
02:53:25.943: 	id: {0.0.0.00000000}.{b20e0f62-2524-4c22-88bb-70fe8d817cb5}
02:53:25.943: ---------------------------------
02:53:25.945: Skipping module '../../obs-plugins/64bit/advanced-scene-switcher-lib.dll', not an OBS plugin
02:53:25.947: [adv-ss] version: 1.21.1
02:53:25.947: [adv-ss] version: 9bedbf6fb826700944c3136f34504cc9a89c39cb
02:53:25.948: [adv-ss] curl loaded successfully
02:53:25.948: [adv-ss] found curl library
02:53:25.970: [adv-ss] attempting to load "C:/Program Files/obs-studio/obs-plugins/64bit/adv-ss-plugins/advanced-scene-switcher-opencv.dll"
02:53:25.972: [adv-ss] successfully loaded "C:/Program Files/obs-studio/obs-plugins/64bit/adv-ss-plugins/advanced-scene-switcher-opencv.dll"
02:53:25.972: [adv-ss] attempting to load "C:/Program Files/obs-studio/obs-plugins/64bit/adv-ss-plugins/advanced-scene-switcher-openvr.dll"
02:53:25.973: [adv-ss] successfully loaded "C:/Program Files/obs-studio/obs-plugins/64bit/adv-ss-plugins/advanced-scene-switcher-openvr.dll"
02:53:25.973: [adv-ss] attempting to load "C:/Program Files/obs-studio/obs-plugins/64bit/adv-ss-plugins/opencv_calib3d460.dll"
02:53:25.973: [adv-ss] successfully loaded "C:/Program Files/obs-studio/obs-plugins/64bit/adv-ss-plugins/opencv_calib3d460.dll"
02:53:25.973: [adv-ss] attempting to load "C:/Program Files/obs-studio/obs-plugins/64bit/adv-ss-plugins/opencv_core460.dll"
02:53:25.973: [adv-ss] successfully loaded "C:/Program Files/obs-studio/obs-plugins/64bit/adv-ss-plugins/opencv_core460.dll"
02:53:25.973: [adv-ss] attempting to load "C:/Program Files/obs-studio/obs-plugins/64bit/adv-ss-plugins/opencv_features2d460.dll"
02:53:25.973: [adv-ss] successfully loaded "C:/Program Files/obs-studio/obs-plugins/64bit/adv-ss-plugins/opencv_features2d460.dll"
02:53:25.973: [adv-ss] attempting to load "C:/Program Files/obs-studio/obs-plugins/64bit/adv-ss-plugins/opencv_flann460.dll"
02:53:25.973: [adv-ss] successfully loaded "C:/Program Files/obs-studio/obs-plugins/64bit/adv-ss-plugins/opencv_flann460.dll"
02:53:25.973: [adv-ss] attempting to load "C:/Program Files/obs-studio/obs-plugins/64bit/adv-ss-plugins/opencv_imgproc460.dll"
02:53:25.973: [adv-ss] successfully loaded "C:/Program Files/obs-studio/obs-plugins/64bit/adv-ss-plugins/opencv_imgproc460.dll"
02:53:25.973: [adv-ss] attempting to load "C:/Program Files/obs-studio/obs-plugins/64bit/adv-ss-plugins/opencv_objdetect460.dll"
02:53:25.973: [adv-ss] successfully loaded "C:/Program Files/obs-studio/obs-plugins/64bit/adv-ss-plugins/opencv_objdetect460.dll"
02:53:25.973: [adv-ss] attempting to load "C:/Program Files/obs-studio/obs-plugins/64bit/adv-ss-plugins/openvr_api.dll"
02:53:25.973: [adv-ss] successfully loaded "C:/Program Files/obs-studio/obs-plugins/64bit/adv-ss-plugins/openvr_api.dll"
02:53:25.986: No AJA devices found, skipping loading AJA UI plugin
02:53:25.986: Failed to initialize module 'aja-output-ui.dll'
02:53:25.987: No AJA devices found, skipping loading AJA plugin
02:53:25.987: Failed to initialize module 'aja.dll'
02:53:25.987: Skipping module '../../obs-plugins/64bit/chrome_elf.dll', not an OBS plugin
02:53:25.988: [CoreAudio encoder]: CoreAudio AAC encoder not installed on the system or couldn't be loaded
02:53:25.989: Failed to load 'en-US' text for module: 'decklink-captions.dll'
02:53:25.989: Failed to load 'en-US' text for module: 'decklink-output-ui.dll'
02:53:25.990: A DeckLink iterator could not be created.  The DeckLink drivers may not be installed
02:53:25.990: Failed to initialize module 'decklink.dll'
02:53:25.991: [Directory watch media] loaded version 0.6.2
02:53:26.175: [AMF] Version 2.7.0 loaded (Compiled: 1.4.14.0, Runtime: 1.4.29.0, Library: 1;4;29;0;22.40.33.01;202302171300;AMD-Radeon-Driver/drivers:9be077d53ce4fc0594d1f2c2f5f6f46b0da8fc24).
02:53:26.238: [AMF] <Id: 1> Unable to set converter transfer characteristic, error AMF_NOT_FOUND (code 11)
02:53:26.301: [AMF] <Id: 2> Unable to set converter transfer characteristic, error AMF_NOT_FOUND (code 11)
02:53:26.303: [AMF] [Capability Manager] Testing Direct3D 11 Adapter 'AMD Radeon RX 6900 XT (VEN_1002/DEV_73bf/SUB_440e1da2/REV_00c0)':
02:53:26.303:   H264/AVC: Supported
02:53:26.303:   H265/HEVC: Supported
02:53:26.303: 
02:53:26.309: Skipping module '../../obs-plugins/64bit/libcef.dll', not an OBS plugin
02:53:26.309: Skipping module '../../obs-plugins/64bit/libEGL.dll', not an OBS plugin
02:53:26.309: Skipping module '../../obs-plugins/64bit/libGLESv2.dll', not an OBS plugin
02:53:26.311: [Move Transition] loaded version 2.8.3
02:53:26.319: [obs-browser]: Version 2.19.0
02:53:26.319: [obs-browser]: CEF Version 103.0.5060.134 (runtime), 103.0.0-5060-shared-textures.2591+g4204d54+chromium-103.0.5060.134 (compiled)
02:53:26.396: [noise suppress]: NVIDIA denoiser disabled, redistributable not found or could not be loaded.
02:53:26.396: Failed to get NVVideoEffects.dll version info size
02:53:26.397: [NVIDIA VIDEO FX]: FX disabled, redistributable not found or could not be loaded.
02:53:26.397: [obs-ndi] obs_module_load: hello ! (version 4.11.1)
02:53:26.397: [obs-ndi] load_ndilib: Trying 'C:/Program Files/NDI/NDI 5 Runtime/v5/Processing.NDI.Lib.x64.dll'
02:53:26.397: [obs-ndi] load_ndilib: Found NDI library at 'C:/Program Files/NDI/NDI 5 Runtime/v5/Processing.NDI.Lib.x64.dll'
02:53:26.400: [obs-ndi] load_ndilib: NDI runtime loaded successfully
02:53:26.400: [obs-ndi] load_ndilib: NDIlib_v5_load found
02:53:26.402: [obs-ndi] obs_module_load: NDI library initialized successfully (NDI SDK WIN64 11:14:50 Feb  2 2023 5.5.3.0)
02:53:26.430: [obs-teleport] version: 0.6.5
02:53:26.433: [obs-websocket] [obs_module_load] you can haz websockets (Version: 5.1.0 | RPC Version: 1)
02:53:26.433: [obs-websocket] [obs_module_load] Qt version (compile-time): 6.3.1 | Qt version (run-time): 6.3.1
02:53:26.433: [obs-websocket] [obs_module_load] Linked ASIO Version: 101201
02:53:26.438: [obs-websocket] [obs_module_load] Module loaded.
02:53:26.442: [Transition Table] loaded version 0.2.6
02:53:26.443: [vlc-video]: Couldn't find VLC installation, VLC video source disabled
02:53:26.443: Failed to load 'en-US' text for module: 'win-capture-audio.dll'
02:53:26.443: [win-capture-audio] Version 2.2.3 (74323e6)
02:53:26.448: ---------------------------------
02:53:26.448:   Loaded Modules:
02:53:26.448:     win-wasapi.dll
02:53:26.448:     win-mf.dll
02:53:26.448:     win-dshow.dll
02:53:26.448:     win-capture.dll
02:53:26.448:     win-capture-audio.dll
02:53:26.448:     vlc-video.dll
02:53:26.448:     transition-table.dll
02:53:26.448:     text-freetype2.dll
02:53:26.448:     rtmp-services.dll
02:53:26.448:     obs-x264.dll
02:53:26.448:     obs-websocket.dll
02:53:26.448:     obs-vst.dll
02:53:26.448:     obs-transitions.dll
02:53:26.448:     obs-text.dll
02:53:26.448:     obs-teleport.dll
02:53:26.448:     obs-qsv11.dll
02:53:26.448:     obs-outputs.dll
02:53:26.448:     obs-ndi.dll
02:53:26.448:     obs-filters.dll
02:53:26.448:     obs-ffmpeg.dll
02:53:26.448:     obs-browser.dll
02:53:26.448:     move-transition.dll
02:53:26.448:     logi_obs_plugin_x64.dll
02:53:26.448:     image-source.dll
02:53:26.448:     frontend-tools.dll
02:53:26.448:     enc-amf.dll
02:53:26.448:     droidcam-obs.dll
02:53:26.448:     dir-watch-media.dll
02:53:26.449:     decklink-output-ui.dll
02:53:26.449:     decklink-captions.dll
02:53:26.449:     coreaudio-encoder.dll
02:53:26.449:     advanced-scene-switcher.dll
02:53:26.449: ---------------------------------
02:53:27.449: ==== Startup complete ===============================================
02:53:27.495: All scene data cleared
02:53:27.495: ------------------------------------------------
02:53:27.505: [win-wasapi: 'MasaÃ¼stÃ¼ Ses'] update settings:
02:53:27.505: 	device id: {0.0.0.00000000}.{d7c56023-8427-448d-ad13-a1535767252c}
02:53:27.505: 	use device timing: 1
02:53:27.505: [Loaded global audio device]: 'MasaÃ¼stÃ¼ Ses'
02:53:27.505: [win-wasapi: 'Mic/Aux'] update settings:
02:53:27.505: 	device id: {0.0.1.00000000}.{c5f5501e-1de6-4817-95be-72f0adde07a0}
02:53:27.505: 	use device timing: 0
02:53:27.506: [noise suppress: 'GÃ¼rÃ¼ltÃ¼ BastÄ±rma'] NVAFX redist is not installed.
02:53:27.506: [Loaded global audio device]: 'Mic/Aux'
02:53:27.506:     - filter: 'GÃ¼rÃ¼ltÃ¼ BastÄ±rma' (noise_suppress_filter_v2)
02:53:27.506:     - filter: 'GÃ¼rÃ¼ltÃ¼ Filtresi' (noise_gate_filter)
02:53:27.506: [Media Source 'Replay Slow Motion']: settings:
02:53:27.506: 	input:                   C:/Users/murdo/Videos/Replay 2023-04-20 00-48-22.mkv
02:53:27.506: 	input_format:            (null)
02:53:27.506: 	speed:                   25
02:53:27.506: 	is_looping:              no
02:53:27.506: 	is_linear_alpha:         no
02:53:27.506: 	is_hw_decoding:          no
02:53:27.506: 	is_clear_on_media_end:   yes
02:53:27.506: 	restart_on_activate:     yes
02:53:27.506: 	close_when_inactive:     no
02:53:27.506: 	ffmpeg_options:          
02:53:27.506: Set FFmpeg options: 
02:53:27.506: [audio monitoring: 'Replay Slow Motion'] audio_monitor_init_wasapi: Failed to get device: 80070490
02:53:27.507: [window-capture: 'Pencere Yakalama'] update settings:
02:53:27.507: 	executable: chrome.exe
02:53:27.507: 	method selected: WGC
02:53:27.507: 	method chosen: WGC
02:53:27.507: 
02:53:27.524: [DroidCamOBS] create: r201 "DroidCam OBS"
02:53:27.525: [DroidCamOBS] checking adb
02:53:27.525: [DroidCamOBS] command not found: adb version
02:53:27.525: [DroidCamOBS] checking ../../data/obs-plugins/droidcam-obs/adb/adb.exe
02:53:27.530: WASAPI: Device 'HoparlÃ¶r (Logitech PRO X Gaming Headset)' [44100 Hz] initialized
02:53:27.533: WASAPI: Device 'Mikrofon (Logitech PRO X Gaming Headset)' [48000 Hz] initialized
02:53:27.575: [DroidCamOBS] iOS USB support not available
02:53:27.575: [DroidCamOBS] activated=1, deactivateWNS=0, is_showing=0, enable_audio=0
02:53:27.575: [DroidCamOBS] video_format=avc video_resolution=640x480
02:53:27.575: [DroidCamOBS] device_info.id=dev_id_wifi device_info.ip=192.168.20.15 device_info.port=4747 device_info.type=1
02:53:27.575: [DroidCamOBS] video_thread start
02:53:27.576: [DroidCamOBS] video_decode_thread start
02:53:27.576: [DroidCamOBS] audio_thread start
02:53:27.580: adding 21 milliseconds of audio buffering, total audio buffering is now 21 milliseconds (source: Mic/Aux)
02:53:27.580: 
02:53:27.588: [Media Source 'Overlay']: settings:
02:53:27.588: 	input:                   C:/Users/murdo/Videos/Replay_Overlay/Replay Overlay.webm
02:53:27.588: 	input_format:            (null)
02:53:27.588: 	speed:                   100
02:53:27.588: 	is_looping:              yes
02:53:27.588: 	is_linear_alpha:         no
02:53:27.588: 	is_hw_decoding:          no
02:53:27.588: 	is_clear_on_media_end:   yes
02:53:27.588: 	restart_on_activate:     yes
02:53:27.588: 	close_when_inactive:     no
02:53:27.588: 	ffmpeg_options:          
02:53:27.588: Set FFmpeg options: 
02:53:27.589: [Media Source 'Replay']: settings:
02:53:27.589: 	input:                   C:/Users/murdo/Videos/Replay 2023-04-20 00-48-22.mkv
02:53:27.589: 	input_format:            (null)
02:53:27.589: 	speed:                   100
02:53:27.589: 	is_looping:              yes
02:53:27.589: 	is_linear_alpha:         no
02:53:27.589: 	is_hw_decoding:          no
02:53:27.589: 	is_clear_on_media_end:   yes
02:53:27.589: 	restart_on_activate:     yes
02:53:27.589: 	close_when_inactive:     no
02:53:27.589: 	ffmpeg_options:          
02:53:27.589: [audio monitoring: 'Replay'] audio_monitor_init_wasapi: Failed to get device: 80070490
02:53:27.589: Set FFmpeg options: 
02:53:27.621: Failed to open file 'C:/Users/murdo/OneDrive/Desktop/Webcam_Masks_Free/Landscape/4-3_Type_4.png': No such file or directory
02:53:27.622: WIC: Failed to create IWICBitmapDecoder from file: C:/Users/murdo/OneDrive/Desktop/Webcam_Masks_Free/Landscape/4-3_Type_4.png
02:53:27.622: gs_image_file_init_internal: Failed to load file 'C:/Users/murdo/OneDrive/Desktop/Webcam_Masks_Free/Landscape/4-3_Type_4.png'
02:53:27.662: [Media Source 'Stinger (Stinger)']: settings:
02:53:27.662: 	input:                   
02:53:27.662: 	input_format:            (null)
02:53:27.662: 	speed:                   100
02:53:27.662: 	is_looping:              no
02:53:27.662: 	is_linear_alpha:         no
02:53:27.662: 	is_hw_decoding:          yes
02:53:27.662: 	is_clear_on_media_end:   yes
02:53:27.662: 	restart_on_activate:     yes
02:53:27.662: 	close_when_inactive:     no
02:53:27.662: 	ffmpeg_options:          
02:53:27.667: Switched to scene '[NS] - Streamlabs'
02:53:27.669: [adv-ss] trying to reconnect to  in 10 seconds.
02:53:27.669: ------------------------------------------------
02:53:27.669: Loaded scenes:
02:53:27.669: - scene 'Oyun':
02:53:27.669:     - source: 'Pencere Yakalama' (window_capture)
02:53:27.669:     - source: 'Oyun Yakala' (game_capture)
02:53:27.669:         - filter: 'Renk DÃ¼zeltme' (color_filter_v2)
02:53:27.669:     - source: 'DroidCam OBS' (droidcam_obs)
02:53:27.669:         - filter: 'GÃ¶rÃ¼ntÃ¼ Maskesi/Blend' (mask_filter_v2)
02:53:27.669:         - filter: 'Kes/KaydÄ±r' (crop_filter)
02:53:27.669:     - source: '[NS] - Streamlabs' (scene)
02:53:27.669: - scene 'Ben':
02:53:27.669:     - source: 'Oyun Yakala' (game_capture)
02:53:27.669:         - filter: 'Renk DÃ¼zeltme' (color_filter_v2)
02:53:27.669:     - source: 'DroidCam OBS' (droidcam_obs)
02:53:27.669:         - filter: 'GÃ¶rÃ¼ntÃ¼ Maskesi/Blend' (mask_filter_v2)
02:53:27.669:         - filter: 'Kes/KaydÄ±r' (crop_filter)
02:53:27.669:     - source: '[NS] - Streamlabs' (scene)
02:53:27.669: - scene 'Tekrar':
02:53:27.669:     - source: 'Oyun' (scene)
02:53:27.669:     - source: '[NS] - Replay' (scene)
02:53:27.669: - scene 'Tekrar AÄŸÄ±r Ã‡ekim':
02:53:27.669:     - source: 'Oyun' (scene)
02:53:27.669:     - source: '[NS] - Replay Slow Motion' (scene)
02:53:27.669: - scene '[NS] - Replay':
02:53:27.669:     - source: 'Replay' (ffmpeg_source)
02:53:27.669:         - monitoring: monitor only
02:53:27.669:         - filter: 'GÃ¶rÃ¼ntÃ¼ Maskesi/Blend' (mask_filter_v2)
02:53:27.669:         - filter: 'Directory watch media' (dir_watch_media)
02:53:27.669:     - source: 'Overlay' (ffmpeg_source)
02:53:27.669: - scene '[NS] - Replay Slow Motion':
02:53:27.669:     - source: 'Replay Slow Motion' (ffmpeg_source)
02:53:27.669:         - monitoring: monitor only
02:53:27.669:         - filter: 'Directory watch media' (dir_watch_media)
02:53:27.669:         - filter: 'GÃ¶rÃ¼ntÃ¼ Maskesi/Blend' (mask_filter_v2)
02:53:27.669:     - source: 'Overlay' (ffmpeg_source)
02:53:27.669: - scene '[NS] - Streamlabs':
02:53:27.669:     - source: 'Chat' (browser_source)
02:53:27.669:     - source: 'TakipÃ§i' (browser_source)
02:53:27.669:         - filter: 'Renk AnahtarÄ±' (color_key_filter_v2)
02:53:27.669:     - source: 'KatÄ±lÄ±mcÄ± SayÄ±sÄ±' (browser_source)
02:53:27.669:         - filter: 'Renk AnahtarÄ±' (color_key_filter_v2)
02:53:27.669:     - source: 'Logo' (browser_source)
02:53:27.669:         - filter: 'Renk AnahtarÄ±' (color_key_filter_v2)
02:53:27.669: ------------------------------------------------
02:53:27.670: [adv-ss] started
02:53:27.670: [adv-ss] try to sleep for 300
02:53:27.671: [adv-ss] current scene:  [NS] - Streamlabs
02:53:27.671: [adv-ss] previous scene: 
02:53:27.971: [adv-ss] condition scene returned 0
02:53:27.971: [adv-ss] Macro Replay returned 0
02:53:27.971: [adv-ss] condition scene returned 0
02:53:27.971: [adv-ss] Macro Replay Slow Motion returned 0
02:53:27.971: [adv-ss] try to sleep for 300
02:53:28.271: [adv-ss] condition scene returned 0
02:53:28.272: [adv-ss] Macro Replay returned 0
02:53:28.272: [adv-ss] condition scene returned 0
02:53:28.272: [adv-ss] Macro Replay Slow Motion returned 0
02:53:28.272: [adv-ss] try to sleep for 300
02:53:28.519: [Media Source 'Replay']: settings:
02:53:28.519: 	input:                   C:/Users/murdo/Videos/Replay 2023-04-20 00-48-22.mkv
02:53:28.519: 	input_format:            (null)
02:53:28.519: 	speed:                   100
02:53:28.519: 	is_looping:              yes
02:53:28.519: 	is_linear_alpha:         no
02:53:28.519: 	is_hw_decoding:          no
02:53:28.519: 	is_clear_on_media_end:   yes
02:53:28.519: 	restart_on_activate:     yes
02:53:28.519: 	close_when_inactive:     no
02:53:28.519: 	ffmpeg_options:          
02:53:28.519: Set FFmpeg options: 
02:53:28.527: [Media Source 'Replay Slow Motion']: settings:
02:53:28.527: 	input:                   C:/Users/murdo/Videos/Replay 2023-04-20 00-48-22.mkv
02:53:28.527: 	input_format:            (null)
02:53:28.527: 	speed:                   25
02:53:28.527: 	is_looping:              no
02:53:28.527: 	is_linear_alpha:         no
02:53:28.527: 	is_hw_decoding:          no
02:53:28.527: 	is_clear_on_media_end:   yes
02:53:28.527: 	restart_on_activate:     yes
02:53:28.527: 	close_when_inactive:     no
02:53:28.527: 	ffmpeg_options:          
02:53:28.527: Set FFmpeg options: 
02:53:28.872: [adv-ss] condition scene returned 0
02:53:28.872: [adv-ss] Macro Replay returned 0
02:53:28.872: [adv-ss] condition scene returned 0
02:53:28.872: [adv-ss] Macro Replay Slow Motion returned 0
02:53:28.872: [adv-ss] try to sleep for 300
02:53:29.172: [adv-ss] condition scene returned 0
02:53:29.172: [adv-ss] Macro Replay returned 0
02:53:29.172: [adv-ss] condition scene returned 0
02:53:29.172: [adv-ss] Macro Replay Slow Motion returned 0
02:53:29.173: [adv-ss] try to sleep for 299
02:53:29.473: [adv-ss] condition scene returned 0
02:53:29.473: [adv-ss] Macro Replay returned 0
02:53:29.473: [adv-ss] condition scene returned 0
02:53:29.473: [adv-ss] Macro Replay Slow Motion returned 0
02:53:29.473: [adv-ss] try to sleep for 300
02:53:46.903: [adv-ss] Macro Replay Slow Motion returned 0
02:53:46.904: [adv-ss] try to sleep for 300
02:53:47.203: [adv-ss] condition scene returned 0
02:53:47.203: [adv-ss] Macro Replay returned 0
02:53:47.203: [adv-ss] condition scene returned 0
02:53:47.203: [adv-ss] Macro Replay Slow Motion returned 0
02:53:47.204: [adv-ss] try to sleep for 300
02:53:47.504: [adv-ss] condition scene returned 0
02:53:47.504: [adv-ss] Macro Replay returned 0
02:53:47.504: [adv-ss] condition scene returned 0
02:53:47.504: [adv-ss] Macro Replay Slow Motion returned 0
02:53:47.504: [adv-ss] try to sleep for 300
02:53:47.804: [adv-ss] condition scene returned 0
02:53:47.804: [adv-ss] Macro Replay returned 0
02:53:47.804: [adv-ss] condition scene returned 0
02:53:47.804: [adv-ss] Macro Replay Slow Motion returned 0
02:53:47.805: [adv-ss] try to sleep for 300

**Comment by WarmUpTill** [comment]
Just adding this comment (from the OBS forums) here for completeness:

Thanks for reporting back that the issue was resolved! :)
And sorry for getting back to you so late!

The only things I noticed in the provided logs / screenshot was:

    There was a sudden time jump in the log that I do not really understand:
    02:53:29.473: [adv-ss] try to sleep for 300
    02:53:46.903: [adv-ss] Macro Replay Slow Motion returned 0
    You seem to have selected to "Wait until transition to target scene is complete".
    Maybe the transition never "ended" or was extremely long and thus the plugin was stuck waiting.

But I guess if everything is working now there is no need to change anything :)
